src/test/java/org/betonquest/betonquest/modules/logger/custom/chat/PlayerPackageFilterTest.java

```diff
@@ -0,0 +1,110 @@
+package org.betonquest.betonquest.modules.logger.custom.chat;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.logging.Level;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * A test for the {@link PlayerPackageFilter}.
+ */
+@SuppressWarnings("PMD.JUnitTestContainsTooManyAsserts")
```

Comment:
While this should not be done in this PR I propose that we increase the limit for this PMD warning to 3 or 4. It is still not enough for these tests, but here it does indeed point to the fact that those test methods test too much.

It might be possible to reduce the assertions in these tests by writing [parametrized tests](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests) that only test one thing but reuse the parameters for multiple tests.

Reply:
i still can not see how i can use [parametrized tests](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests) here. May we improve this later if you know a way